src/App.css

```diff
@@ -1,7 +1,3 @@
-.App {
-  text-align: center;
-}
-
 .App-logo {
```

Comment:
- [ ] Please, consider that these staying are not used anywhere in your code. It will be better to remove this styling 👍